packages/flutter/lib/src/cupertino/picker.dart

```diff
@@ -321,20 +319,20 @@ class _CupertinoPickerState extends State<CupertinoPicker> {
 // scroll controller.
 class _CupertinoPickerSemantics extends SingleChildRenderObjectWidget {
   const _CupertinoPickerSemantics({
-    Key key,
-    Widget child,
-    @required this.scrollController,
+    Key? key,
+    Widget? child,
+    required this.scrollController,
   }) : super(key: key, child: child);
 
   final FixedExtentScrollController scrollController;
 
   @override
-  RenderObject createRenderObject(BuildContext context) => _RenderCupertinoPickerSemantics(scrollController, Directionality.of(context));
+  RenderObject createRenderObject(BuildContext context) => _RenderCupertinoPickerSemantics(scrollController, Directionality.of(context)!);
```

Comment:
We should assert here that Directionality is not null with `debugCheckHasDirectionality` in an assert. That will give people a nicer error message then a failed `!` check.

Reply:
Done